### PR TITLE
Loading constants in initializers/active_record_extensions.rb

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: config/initializers/active_record_extensions.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
+require 'host_mix'
+require 'has_many_common'
+require 'strip_whitespace'
+require 'parameterizable'
+
 class ActiveRecord::Base
```
Comment (tbrisker) [inline config/initializers/active_record_extensions.rb]: TBH, this initializer predates the introduction of ApplicationRecord in rails. Probably most if not all of the extensions actually belong there.

File: app/models/application_record.rb [hunk 2]
```diff
@@ -1,7 +1,16 @@
 require_relative 'concerns/audit_associations'
+require_relative 'concerns/host_mix'
+require_relative 'concerns/has_many_common'
+require_relative 'concerns/strip_whitespace'
+require_relative 'concerns/parameterizable'
```
Comment (ezr-ondrej) [inline app/models/application_record.rb]: `concerns` subfolders are in a autoload path, so you should not need this, it should work by leaving it out to autoloader to load those. Being in autoload_path means they are roots and `concerns/host_mix` => `HostMix` is expected by autoloader.

I believe the previous require is here by mistake and can be removed while we are at it.

**Review by ezr-ondrej — CHANGES_REQUESTED** [review]
Apart of the requires, big :+1:
It's a nice cleanup :)

**Review by ezr-ondrej — APPROVED** [review]
Thank you @stejskalleos ! :+1:

**Comment by lzap** [comment]
Why Hostmix was moved from `Host::Hostmix` into the root namespace? It broke foreman_setup for no reason. Just wondering if we could keep the namespace. Anyway, thanks for driving the change :-)

I fixed foreman_setup.

**Comment by ezr-ondrej** [comment]
> Why Hostmix was moved from `Host::Hostmix` into the root namespace? It broke foreman_setup for no reason. Just wondering if we could keep the namespace. Anyway, thanks for driving the change :-)

I could have stayed in the namespace, but it was not too logical, as `Host` would suggest it's a Host extension, which it is not.
Tho sorry for disturbing plugins again, this one I've even checked, but didn't find any uses.

**Pull request merged**